services/API-service/src/shared/helper.service.ts

```diff
@@ -94,8 +94,9 @@ export class HelperService {
     countryCodeISO3: string,
     disasterType: DisasterType,
   ): Promise<DateDto> {
-    const triggerPerLeadTimeRepository =
-      this.dataSource.getRepository(TriggerPerLeadTime);
+    const triggerPerLeadTimeRepository = this.dataSource.getRepository(
```

Comment:
`const alertPerLeadTimeRepository = this.dataSource.getRepository(AlertPerLeadTimeEntity);`

Reply:
same